### DISCUSSION TITLE
Dashboard - Hide original description

### DISCUSSION THREAD

**Discussion by roberthaller** [firefly-iii/firefly-iii#8890] [Firefly III] [answered]
Hello everyone,
I've got a question regarding changes made by rules and/or audit-log. 

I'm using the firefly API to add transactions from different sources (csv, xlsx, etc.). Works like a charm, however, the data contained in those files is not tagged and categorized nor easily associated to specific accounts. Therefore I've created rules to adjust description, accounts, etc..

**Question**:

The **dashboard** shows both the original as well as the new **description** (applied by a rule). Since the original might be very long, I would like to know if it is possible to **hide** the **original description** or if I've other possibilities already available in firefly?

Kind regards,
Robert

**Comment by JC5** [comment]
Do you have a screenshot that shows what you mean? Firefly III doesn't save the original description, as far as I know.

**Reply by roberthaller** [reply, marked as answer]
Thanks for the fast response.

Since it wasn't clear what I meant, I got curious.
I looked a bit deeper and recognized where the 'additional' text comes from:

The dashboard shows **Group: Description** (didn't know that)
![image](https://github.com/firefly-iii/firefly-iii/assets/66618936/30f6bc2f-7191-46cf-b7a3-4d57946dd33f)

So what I thought to be the "original description" was actually the **group title**. 

The API implementation I used automatically applies the description of the first split to the group title and therefore it seemed like a "original description" was shown. No issue/problem after all.

Sorry for the inconvenience and thanks again for the fast reply.
